docs/userGuide/syntax/popovers.mbdf

```diff
@@ -79,8 +79,8 @@ This is the same <trigger for="pop:trigger_id">trigger</trigger> as last one.
 Name | Type | Default | Description
 ---- | ---- | ------- | ------
 trigger | `String` |	`hover`	| How the Popover is triggered.<br>Supports: `click`, `focus`, `hover`.
-header{{slot_icon_trigger}} | `String` | `''` | Popover header, supports inline markdown text.
-content{{slot_icon_trigger}} | `String` | `''` | Popover content, supports inline markdown text.
+header{{slot_icon_trigger}} | `String` | `''` | Popover header, supports MarkBind syntax.
+content{{slot_icon_trigger}} | `String` | `''` | Popover content, supports MarkBind syntax.
```

Comment:
Think we could also allow/fix the typo now that reactivity is possible
```suggestion
content{{slot_info_trigger}} | `String` | `''` | Popover content, supports MarkBind syntax.
```

Reply:
Thanks, I didn't realise this!